paddle/operators/math/context_project.cu

```diff
@@ -22,6 +22,9 @@ namespace math {
 
 template class ContextProjectFunctor<platform::GPUPlace, float>;
 template class ContextProjectFunctor<platform::GPUPlace, double>;
+template class ContextProjectFunctor<platform::GPUPlace, int>;
+template class ContextProjectFunctor<platform::GPUPlace, int64_t>;
+template class ContextProjectFunctor<platform::GPUPlace, bool>;
```

Comment:
`ContextProjectFunctor` is not exposed to user, it is used in `sequence_conv_op`. It seems there is no need to define int, int64_t, bool type for this functor.

Add, [`sequence_conv_op`](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/sequence_conv_op.cc) needs to register double type.

Reply:
Done. make `sequence_conv_op` supports multiple precision.